### PR TITLE
Services/SOC: Added command headers to some of the soc commands.

### PR DESCRIPTION
**Pull request by Subv**
These were found by reverse engineering the soc binary

### CODE REVIEW

**Comment by purpasmart96** [comment]
LGTM

**Comment by archshift** [comment]
LGTM

File: src/core/hle/service/soc_u.cpp [hunk 1]
```diff
@@ -460,8 +463,10 @@ static void Accept(Service::Interface* self) {
         Memory::WriteBlock(cmd_buffer[0x104 >> 2], (const u8*)&ctr_addr, max_addr_len);
     }
 
-    cmd_buffer[2] = ret;
+    cmd_buffer[0] = IPC::MakeHeader(4, 2, 2);
     cmd_buffer[1] = result;
+    cmd_buffer[2] = ret;
+    cmd_buffer[3] = IPC::StaticBufferDesc(static_cast<u32>(max_addr_len), 0);
 }
```
Comment (yuriks) [inline src/core/hle/service/soc_u.cpp]: Where's `cmd_buff[4]`?

Comment (Subv) [inline src/core/hle/service/soc_u.cpp]: I was unable to find what it was being set to, so i left it out

**Comment by bunnei** [comment]
Lgtm

**Pull request merged**